pages/community-lessons.md

```diff
@@ -22,69 +22,94 @@ permalink: "/community-lessons/"
 or as complements to a "standard" 2-day Carpentries workshop.
 These lessons can also be used by independent learners, outside of workshops.
 
 ## The Carpentries Incubator
 
 The Carpentries Incubator is a place to share Carpentries-style teaching
 materials at all stages of development, to collaborate on lesson development,
-and receive feedback from other community members.
+and receive feedback from other Carpentries community members.
 
 Lessons in The Carpentries Incubator are developed and supported by community
-members and are not officially endorsed by The Carpentries or HPC Carpentry. We
+members and are not officially endorsed by The Carpentries (or HPC Carpentry
+unless they are part of our curriculum). We
 encourage you to browse the Incubator lessons for materials that meet your
 needs and to use these materials freely (all lessons should be licensed 
 [CC-BY 4.0](https://creativecommons.org/licenses/by/4.0/)).
 
-If you are interested in developing or submitting a lesson to the HPC Carpentry
+If you are interested in developing or submitting a lesson to the Carpentries
 Incubator, [contact us]({{ site.contact }}).
 Please read the information in The Carpentries' [Development of Lessons page](
 https://carpentries.org/involved-lessons/) if you would like to contribute to
 the development of a lesson already present in The Carpentries Incubator.
 
-## Lessons in The Carpentries Incubator
+### _Lessons in The Carpentries Incubator_:
+
+<br>
 
-{% assign lesson_list = site.data.community_lessons | where: "hpc_carpentry_org","the-carpentries-incubator" %}
+{% assign lesson_list = site.data.hpc_lessons | where: "carpentries_org","carpentries-incubator" %}
 {% include lesson_table %}
 
 <hr>
 
-## HPC Carpentry Lab
+## HPC Carpentry
 
-The HPC Carpentry Lab is a place for sharing high-quality, peer-reviewed lessons
+The [HPC Carpentry GitHub organisation](https://github.com/hpc-carpentry) is a
+place for sharing HPC-oriented, high-quality, peer-reviewed lessons
 that follow best practices in pedagogy and the general teaching practices used in
 Carpentries workshops.
 
-Lessons in HPC Carpentry Lab have been peer-reviewed.
-We encourage you to browse the Lab lessons for materials that meet your needs and
+Lessons in HPC Carpentry have been peer-reviewed and each lesson includes
+an indication of the level of maturity of the content.
+We encourage you to browse the lessons for materials that meet your needs and
 to use these materials freely (all lessons are
 licensed [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/)).
 
-<!--At this time, we are not accepting lesson submissions to HPC Carpentry Lab. If
-you are interested in having a lesson reviewed for inclusion in the Lab, please
-submit it first to the Incubator.
+### _Lessons under HPC Carpentry_:
+
+<br>
+
+{% assign lesson_list = site.data.hpc_lessons | where: "carpentries_org","hpc-carpentry" %}
+{% include lesson_table %}
+
+<hr> 
+
+## HPC Carpentry Listings
```

Comment:
If this starts getting populated, it should really be broken off into a separate page

Reply:
I have a lesson I would like to include here